engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -1092,7 +1130,7 @@ namespace dmGameSystem
 
     static inline dmGraphics::HVertexDeclaration GetBaseVertexDeclaration(ModelWorld* world, dmRender::HMaterial material, bool has_skin_data)
     {
-        return has_skin_data && dmRender::GetMaterialHasSkinnedAttributes(material) ? world->m_VertexDeclarationSkinned : world->m_VertexDeclaration;
+        return has_skin_data ? world->m_VertexDeclarationSkinned : world->m_VertexDeclaration;
```

Comment:
We need to use the skinned vertex declaration even if the material doesn't have skinned attributes. We will only bind the streams from the declaration that exist anyway.